docs/guidelines.md

```diff
@@ -0,0 +1,200 @@
+# Actions Guidelines
+
+> Last updated: **April 24th, 2025**
```

Comment:
git will tell us the date the file was last modified.  Do we want a modification date here where someone will have to remember to update it?

(and just to complain about git, cvs would handle this case just fine using `$Date$` :) )

Reply:
Good point, will remove.